### ISSUE TITLE
Port to OSX?

### ISSUE THREAD

**Issue by saturnism** [mastahyeti/security-key#6]
Would love to see this project ported to OSX w/ the recent Macbook Pro + TouchID!

**Comment by btoews** [comment]
Agreed. I started working on that, but upgrading things to Swift 3 turned out to be messy. A PR would be welcome.
